modules/kmm-configuring-kmmo.adoc

```diff
@@ -49,6 +51,9 @@ worker:
 | `healthProbeBindAddress`
 | Defines the address on which the Operator monitors for kubelet health probes. The recommended value is `:8081`.
 
+|`job.gcDelay`
+|Defines the duration for which successful build pods should be preserved before they are deleted. There is no recommended value for this setting. For information on the valid values for this setting, see link:https://pkg.go.dev/time#ParseDuration[ParseDuration].
```

Comment:
* Per ISG, should be "information about"
* Per ISG, you don't have to reword to use the "for which" construction if it makes the sentence read a bit awkwardly. Recommend rewording
* And just a question if it's necessary to say "There is no recommended value for this setting." I know you put recommended values in the others, but I would assume that saying nothing would mean the same thing and would be more concise. But also fine if you feel like it's necessary.

```suggestion
|Defines the duration that successful build pods should be preserved for before they are deleted. There is no recommended value for this setting. For information about the valid values for this setting, see link:https://pkg.go.dev/time#ParseDuration[ParseDuration].
```

Reply:
Made these changes.